src/dockerfiles/lambda/function/function.py

```diff
@@ -0,0 +1,40 @@
+import base64
```

Comment:
I'm curious if this essentially does the same as https://github.com/aqueducthq/aqueduct/blob/main/src/python/aqueduct_executor/start-function-executor.sh and they should eventually merge